core/lib/multivm/src/tracers/validator/mod.rs

```diff
@@ -133,7 +139,11 @@ impl<H> ValidationTracer<H> {
         // The user is allowed to touch its own slots or slots semantically related to him.
         let valid_users_slot = address == self.user_address
             || u256_to_account_address(&key) == self.user_address
-            || self.auxilary_allowed_slots.contains(&u256_to_h256(key));
+            || self
+                .auxilary_allowed_slots
+                .range(u256_to_h256(key - Self::MAX_ALLOWED_SLOT_OFFSET)..=u256_to_h256(key))
```

Comment:
What will happen if this is underflow?

Reply:
I thought that this won't matter since probability of getting a keccak < uint256(127) is basically 0, but I see now that it underflow can happen regardless. Will fix